requirements.txt

```
@@ -1,7 +1,5 @@
-EGCG-Core>=0.11
-jinja2==2.8
+EGCG-Core==0.11.1
```

That should be the newer version 
```suggestion
EGCG-Core==0.11.2
```